src/QsCompiler/Tests.Compiler/TestCases/QirTests/TestBigInts.qs

```diff
@@ -16,5 +16,8 @@ namespace Microsoft.Quantum.Testing.QIR
     @EntryPoint()
     function Main() : Unit {
         let _ = TestBigInts(0L, 0L);
+        let N1 = 12345L;
```

Comment:
It looks like the code generation only checks the generated IR for TestBigInts. Could you hence either add these bigint instantiations to the TestBigInts function instead, or add a file to also check the generated .ll code for the Main function?

Reply:
Added another file to test Main. Changed qirTest to qirMultiTest.